api.js

```diff
@@ -132,11 +132,8 @@ Api.prototype.observationUpdate = function (req, res, m) {
         })
         opts.links = old.id
       }
-      var newObs = Object.assign(old, {
-        properties: obs.properties,
-        lat: obs.lat,
-        lon: obs.lon
-      })
+      var newObs = Object.assign(old, m, {})
+      delete newObs.version
```

Comment:
Could you delete `newObs.id` as well?

Reply:
Oh, and add a test that sets both & ensures they aren't overriding the true version/id?